src/main/kotlin/rules/NewAlbum.kt

```diff
@@ -0,0 +1,30 @@
+package com.eigenholser.flac2mp3.rules
+
+import com.eigenholser.flac2mp3.AlbumFact
+import com.eigenholser.flac2mp3.AlbumRule
+import com.eigenholser.flac2mp3.states.AlbumStates
+import com.eigenholser.flac2mp3.states.NewAlbumEvent
+import org.jeasy.rules.api.Facts
+import org.jeasy.states.api.FiniteStateMachine
+
+class NewAlbum(private val albumStateMachine: FiniteStateMachine): ConversionRule {
```

Comment:
This class can be implemented without ConversionRule, it may be better to simply implement Rule and override compareTo.